pkg/server/server.go

```diff
@@ -318,9 +318,11 @@ func (s *Server) ListObjects(ctx context.Context, req *openfgav1.ListObjectsRequ
 	))
 	defer span.End()
 
+	const methodName = "listObjects"
```

Comment:
Here and below, can we follow existing conventions so as to not increase the cognitive impact due to the cardinality of different label values?

Existing labels for `grpc_method` and `method` use `listobjects`, `check`, `streamedlistobjects`, etc.. For example,

<img width="917" alt="Screenshot 2023-08-30 at 2 22 28 PM" src="https://github.com/openfga/openfga/assets/2899204/fd49e440-f3b4-4b41-a47e-4b71d97aab1d">